www/app/DeviceTimers.js

```diff
@@ -0,0 +1,837 @@
+                } else {
+                    vm.week.type = 'SelectedDays';
+                    vm.week.days.mon = !!(value & 0x01);
+                    vm.week.days.tue = !!(value & 0x02);
+                    vm.week.days.wed = !!(value & 0x04);
+                    vm.week.days.thu = !!(value & 0x08);
+                    vm.week.days.fri = !!(value & 0x10);
+                    vm.week.days.sat = !!(value & 0x20);
+                    vm.week.days.sun = !!(value & 0x40);
+                }
+            });
+        }
+
+        function initLEDLightSettings() {
+            var $picker = $element.find('#picker');
+            var $whiteSlider = $element.find('#ledtable #white_slider');
+
+            vm.lightSettings = {
+                hue: 128,
+                s: 180,
+                brightness: 100,
+                whiteLevel: 0,
+                isWhite: false
+            };
+
+            $picker.colpick({
+				flat: true,
+				layout: 'hex',
+				submit: 0,
+                color: {
+                    h: vm.lightSettings.hue,
+                    s: vm.lightSettings.s,
+                    b: vm.lightSettings.brightness
+                },
+				onChange: function (hsb, hex, rgb, el, fromSetColor) {
+					if (!fromSetColor) {
+                        vm.lightSettings.hue = hsb.h;
+                        vm.lightSettings.brightness = hsb.b;
+                        vm.lightSettings.s = hsb.s;
+                        vm.lightSettings.isWhite = (hsb.s < 20);
+                        $scope.$apply();
+					}
+				}
+            });
+            
+            $whiteSlider.slider({
+                range: "min",
+				min: 1,
+				max: 255,
+                value: 0,
+                slide: function (event, ui) {
+                    vm.lightSettings.whiteLevel = ui.value-1;
+                    $scope.$apply();
+                },
+                stop: function (event, ui) {
+                    vm.lightSettings.whiteLevel = ui.value-1;
+                    $scope.$apply();
+                }
+            });
+
+            $scope.$watch('vm.lightSettings', function(newValue, oldValue) {
+                $picker.colpickSetColor({
+                    h: vm.lightSettings.hue,
+                    s: vm.lightSettings.s,
+                    b: vm.lightSettings.brightness
+                });
+
+                if (oldValue.whiteLevel != newValue.whiteLevel) {
+                    $whiteSlider.slider('value', vm.lightSettings.whiteLevel);
+                }
+
+                setColorDebounce();
+            }, true)
+        }
+
+        function initDatePicker() {
+            var nowTemp = new Date();
+            var now = new Date(nowTemp.getFullYear(), nowTemp.getMonth(), nowTemp.getDate(), 0, 0, 0, 0);
+            var element = $element.find("#sdate");
+
+            element.datepicker({
+                minDate: now,
+                defaultDate: now,
+                dateFormat: "mm/dd/yy",
```

Comment:
You should never hard-code the US-local "mm/dd/yy" format. Please use the ISO standard YYYY-MM-DD. That's what Domoticz already uses in *almost* all of its user interface; please don't add new exceptions that make it inconsistent.
I've updated #1973  to fix this.